backend/flask_api.py

```diff
@@ -30,6 +29,74 @@
 ALLOWED_EXTENSIONS = {'txt', 'json', 'json5'}
 app.config['UPLOAD_FOLDER'] = ''
 app.config['MAX_CONTENT_PATH'] = 1000000  # 100.000 byte = 100kb
+app.config['MONGODB_DB'] = 'knexDB'
+app.config['MONGODB_HOST'] = 'mongodb'
+app.config['MONGODB_PORT'] = 27017
+
+es = Elasticsearch([{'host': 'elasticsearch', 'port': 9200}])
+
+client = MongoClient('mongodb:27017')
```

Comment:
had to do that for the MongoEngine, but yes I agree, a good idea to use it for the project db client as well

would look like an abomination though, I guess there's no direct constructor